src/query_builder.ts

```diff
@@ -1,6 +1,12 @@
 import { ScopedVars } from '@grafana/data';
+import { TemplateSrv } from '@grafana/runtime';
 
-export default function interpolateKustoQuery(query: string, scopedVars?: ScopedVars): string {
+export default function interpolateKustoQuery(
+  query: string,
+  templateSrv: TemplateSrv,
+  interpolateVariable: (value: any, variable: any) => string | number,
```

Comment:
nit: you can save one parameter if you pass a function: `replace: (val: string) => string` and call it with: `(val: string) => templateSrv.replace(p2, scopedVars, interpolateVariable)`